packages/test-app-frontend/src/App/ITwinJsApp/AppUi/redux/redux-ts.ts

```diff
@@ -1,9 +1,3 @@
-/*---------------------------------------------------------------------------------------------
-* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
```

Comment:
We probably want to keep the copyright. All files should have it.

Reply:
This code looked suspiciously different, so I made an assumption it was not written by us. [But it probably was](https://github.com/iTwin/itwinjs-core/blob/release/3.7.x/ui/appui-react/src/appui-react/redux/redux-ts.ts). Reinstated the header in b9b78a12419e6851f94a70fc23e6208084e879af.